### PR TITLE
Add support for deploying on Debian systems

### PR DESCRIPTION
**Pull request by OddBloke**

### CODE REVIEW

**Review by bliemli — COMMENTED** [review]
Nice! Looks good to me. However, I'd rather @fabaff had a quick look at it as well. I don't want to rush into my first code approval in a home-assistant repository without it being cross-checked.

**Comment by OddBloke** [comment]
I've got some more PRs to submit after this one, so a merge would be nice. :-)

**Comment by bliemli** [comment]
Please add Debian and the corresonding versions as supported platform in the meta file, then this is good to go imo.

**Comment by OddBloke** [comment]
@bliemli Done!

**Review by bliemli — APPROVED** [review]
Thanks!

File: tasks/preparation.yml
Comment (fabaff) [inline tasks/preparation.yml:22]: Isn't `python2-dnf` pulling in `python2`? Well, better be safe than sorry.

**Review by fabaff — APPROVED** [review]
Let's merge it.

**Pull request merged**
